.github/workflows/release.yml

```diff
@@ -163,15 +163,6 @@ jobs:
           check-name: 'Oclif Release'
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           wait-interval: 10
-      - name: Run sanity testing on release-draft
```

Comment:
We need to verify that brew testing will happen fine for draft release after these changes